tests/S3/ObjectCopierTest.php

```diff
@@ -431,14 +440,14 @@ public function testS3ObjectCopierMultipartDoesTransformUnicodeKeyEncodedToURL()
 
         $uploader = new ObjectCopier(
             $client,
-            ['Bucket' => 'sourceBucket', 'Key' => $chinese],
-            ['Bucket' => 'bucket', 'Key' => $chinese],
+            ['Bucket' => 'sourceBucket', 'Key' => $input],
+            ['Bucket' => 'bucket', 'Key' => $input],
             'private'
         );
 
         $this->assertFalse($this->mockQueueEmpty());
         $result = $uploader->copy();
 
-        $this->assertEquals($url, $result['ObjectURL']);
+        $this->assertEquals($result['ObjectURL'], "https://bucket.s3.amazonaws.com/".$expectedOutput);
```

Comment:
Style nitpick: `"https://bucket.s3.amazonaws.com/{$expectedOutput}"` would be a bit cleaner.